lm-agent/lm_agent/backend_utils/models.py

```diff
@@ -79,13 +79,15 @@ class JobSchema(BaseModel):
 
     bookings: List[BookingSchema] = []
 
+    model_config = ConfigDict(from_attributes=True)
```

Comment:
Why does this need a `from_attributes=True`? Shouldn't this only be set if `orm_mode` was set before?

Reply:
`orm_mode` doesn't exist in pydantic v2, instead, just setting `from_attributes` should work ([reference](https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.from_attributes))

Reply:
But this schema didn't have the `orm_mode` set before, since the agent doesn't interface with the database.
It doesn't need to have `from_attributes=True`, the same way the other schemas in the same module don't have it.

Reply:
According to [Pydantic's documentation](https://docs.pydantic.dev/latest/concepts/models/#arbitrary-class-instances):

> "Pydantic models can also be created from arbitrary class instances by reading the instance attributes corresponding to the model field names. One common application of this functionality is integration with object-relational mappings (ORMs)."

Here we're not creating it from a class. Notice that all instances of `model_validate` is converting a response from the API (JSON converted to a `dict`) into a Pydantic model. All the other schemas (ConfigurationSchema, FeatureSchema and LicenseBookingRequest) are doing the same without `from_attributes=True` being set.

Reply:
It is true that you do not need `from_attributes=True` to use `model_validate`.

Reply:
adjusted in [55c783f](https://github.com/omnivector-solutions/license-manager/pull/357/commits/55c783f65a6ffc5d310636f2eca6c6f8013e74fa)